test/blackbox-tests/test-cases/pkg/git-source.t

```diff
@@ -0,0 +1,26 @@
+Test fetching from git
+
+  $ mkdir somerepo
+  $ cd somerepo
+  $ git init --quiet
+  $ echo "hello world" > foo
+  $ git add foo
+  $ git commit -am _ --quiet
+  $ cd ..
+
+  $ MYGITREPO=$PWD/somerepo
+
+  $ mkdir foo && cd foo
+  $ mkdir dune.lock
+  $ cat >dune.lock/lock.dune <<EOF
+  > (lang package 0.1)
+  > EOF
+  $ cat >dune.lock/test <<EOF
+  > (source (fetch (url "git+file://$MYGITREPO")))
```

Comment:
@Leonidas-from-XIV what's the right syntax to fetch from a git repo  that's on the file system?